deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+echo "$(basename $0)"
+set -x
```

Comment:
* `set -x` looks good when you debug your scrips but it is excessive when you use your scripts on production. Remove it

* add `set -euo pipefail` after shebang